PowerUp/app/src/main/res/layout/gamemap.xml

```diff
@@ -39,9 +31,9 @@
         android:id="@+id/BoyfriendButton"
         android:layout_width="@dimen/map_click_width"
         android:layout_height="@dimen/map_click_height"
-        android:layout_above="@id/ContinueButtonMapGameBegins"
         android:layout_alignLeft="@id/HouseButton"
         android:layout_alignStart="@id/HouseButton"
+        android:layout_alignTop="@+id/SchoolButton"
```

Comment:
No actually, the school button s being declared after the button which
requires its reference so I need to use @+Id for school button, one
exception.
On 22-Oct-2015 1:07 PM, "Alma Castillo Antolin" <notifications@github.com>
wrote:

> In PowerUp/app/src/main/res/layout/gamemap.xml
> <https://github.com/systers/powerup-android/pull/4#discussion_r42718691>:
>
> >          android:layout_alignLeft="@id/HouseButton"
> >          android:layout_alignStart="@id/HouseButton"
> > +        android:layout_alignTop="@+id/SchoolButton"
>
> You forgot to change it to @id here, since you are not declaring a new
> id, but referencing one
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/powerup-android/pull/4/files#r42718691>.
>